.github/workflows/check-administration.yaml

```diff
@@ -0,0 +1,38 @@
+name: 'Check Administration'
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - 'administration/**'
+jobs:
+  check-administration:
```

Comment:
💭  I would reorder the naming here: `administration-check`

This way, related jobs group together visually when we have more than one job per high level module.

The same principle would also apply to the file name.